puppet/modules/web/manifests/stats.pp

```diff
@@ -19,13 +32,24 @@
     webroot_paths => [$webroot],
   }
 
-  apache::vhost { 'shiny_server':
-    port          => '80',
-    servername    => $hostname,
-    docroot       => $webroot,
-    docroot_owner => $::apache::user,
-    docroot_group => $::apache::group,
-    proxy_pass    => $proxy_pass,
+  if $https_only {
```

Comment:
Maybe we should unify this with `$https`? I don't see a reason to allow `https` but not run `https_only` these days.

Reply:
Yeah, fair point. Fixed.